maths/Kadane's_algorithm.py

```diff
@@ -1,11 +1,26 @@
 """
     Kadane's algorithm to get maximum subarray sum
-    Please enter the decimal values only and use spaces as seperator
+    Please enter the decimal values only and use spaces as separator
+"""
+
+
+"""
+Doc-Test :-
+ 
+Enter the element of array with spaces  :-
+>>> 2 3 -9 8 -2 
+Maximum subarray sum of  [2, 3, -9, 8, -2] is :- 8
+
+
+Enter the element of array with spaces  :-
+>>> -9 -8 -7 -6 -2
+Maximum subarray sum of  [-9, -8, -7, -6, -2] is :- -2
+
 """

```

Comment:
Doctest will not run if it is not in a docstring.  The second multiline string is not a docstring but the first is.  Please move these tests into the _docstrings of the functions_.  Please run `python3 -m doctest -v Kadane's_algorithm.py` on your own computer to make sure that they run and pass.